build/bootstrap.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+###
+# This script handles provisioning of the Delivery Optimization client components on supported platforms
+###
+
+# bootstrap scripts will exit immediately if a command exits with a non-zero status
```

Comment:
can remove this comment

Reply:
Maybe keep it and shorten it to 'Treat all command failures as fatal'